hash_practice/exercises.py

```diff
@@ -1,19 +1,56 @@
 
+from http.client import CONTINUE

```

Comment:
It looks like this unneeded `import` statement got pulled in by an overzealous IDE. It's usually worth removing unnecessary `import` statements for the sake of maintaining clean code for style purposes.